incubator/kafka/notes.txt

```diff
@@ -0,0 +1,36 @@
+You can connect to Kafka by running a simple pod in the K8s cluster like this with a configuration like this:
+
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: testclient
+    namespace: kafka
+  spec:
+    containers:
+    - name: kafka
+      image: solsson/kafka:0.10.0.1
+      command:
+        - sh
+        - -c
+        - "exec tail -f /dev/null"
+
+  Once you have the testclient pod above running, you can list all kafka
+  topics with:
+
+  kubectl -n kafka exec testclient -- ./bin/kafka-topics.sh --zookeeper my-release-zk:2181 --list
+
+  To create a new topic:
+  kubectl -n kafka exec testclient -- ./bin/kafka-topics.sh --zookeeper my-release-zookeeper-zk:2181 --topic test1 --create --partitions 1 --replication-factor 1
+
+  To listen for messages on a topic:
+  kubectl -n kafka exec -ti testclient -- ./bin/kafka-console-consumer.sh --zookeeper my-release-zookeeper-zk:2181 --topic test1 --from-beginning
+
+  To stop the listener session above press: Ctrl+C
+
+  To start an interactive message producer session:
+  kubectl -n kafka exec -ti testclient -- ./bin/kafka-console-producer.sh --broker-list my-release-broker-kf.kafka.svc.cluster.local:9092 --topic test1
```

Comment:
You can simplify the dns name there and make it `--broker-list {{ .Release.Name }}-kf:9092`. Again this make the command work across namespaces.